README.md

```diff
@@ -225,10 +225,10 @@ metadata:
 ```
 
 Next, deploy four example apps that demonstrate how `ResourceClaim`s,
-`ResourceClaimTemplate`s, and custom `ClaimParameter` objects can be used to
-request access to resources in various ways:
+`ResourceClaimTemplate`s, and custom `GpuConfig` objects can be used to
```

Comment:
This is more of a general comment, but since the resource driver is not actually GPU-specific we may want to drop the `GPU*` nomenclature in some follow-up. Not a blocker here.

Reply:
What do you mean? We agreed on writing an example GPU driver from the manufacturer "example.com". Everything about the driver is prefaced on this assumption.